Modules/Platform/Windows-MSVC.cmake

```diff
@@ -233,8 +233,11 @@ else()
     set(CMAKE_C_STANDARD_LIBRARIES_INIT "kernel32.lib user32.lib gdi32.lib winspool.lib comdlg32.lib advapi32.lib shell32.lib ole32.lib oleaut32.lib uuid.lib odbc32.lib odbccp32.lib")
   endif()
 
-  if(MSVC_VERSION GREATER_EQUAL 1915)
-    set(_JMC "/JMC")
+  # JMC only supported in MSVC
+  if("x${CMAKE_C_COMPILER_ID}" STREQUAL "xMSVC" OR "x${CMAKE_CXX_COMPILER_ID}" STREQUAL "xMSVC" )
```

Comment:
there is no need at all of the "x" suffix, as the policy CMP0054 is already deprecated since CMake 3.1. I would simply remove it.

Reply:
what about CLI/managed C++? my understanding is that CLI is not compatible with JMC

Reply:
@paulmaybee please have a look as well

Reply:
> there is no need at all of the "x" suffix, as the old behavior of policy CMP0054 is already deprecated since CMake 3.1. I would simply remove it.

I spent some time debugging this.  This pattern is needed, policies like CMP0054 don't apply if the caller cmakelists specifies an older version of cmake (e.g. 2.x) which happens in bullet codebase.  Otherwise "MSVC" will be substituted with the variable value of same name even if quoted etc.

> what about CLI/managed C++? my understanding is that CLI is not compatible with JMC

I checked into this.  CLI/managed C++ is supported by VS code gen only so not addressing in this check-in.